maquetado/scss/navbar.scss

```diff
@@ -0,0 +1,21 @@
+.navbar {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0px 10px 13px -7px #000000, 0px 7px 5px 0px rgba(125,125,125,0.43);
+  display: flex;
+  height: $nav-height;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  z-index: 1;
+}
+
+.navbar-logo {
+  height: $nav-logo-height;
+  margin-left: 20px;
```

Comment:
I would set `margin` here, you have separated it from the left border, but what if you need to add another element beside this logo?